### ISSUE TITLE
When I continue a training it starts with much worse performance then best.pt from last run

### ISSUE THREAD

**Issue by haimat** [ultralytics/yolov5#9234]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### YOLOv5 Component

Training

### Bug

I have trained a YOLOv5 model with a mAP@0.5:0.95 performance of 0.53, the following line shows this mAP@0.5:0.95 value over training:

![image](https://user-images.githubusercontent.com/6633976/187660188-b7a23cfc-8336-4afa-bcf6-dc9cc751c3dd.png)

This curve led to the assumption that performance could be even better, so started another training run, based on the `best.pt` file from the first run. However, after only 2-3 epochs the model performance became much worse, it went down to under 0.1, only to get little better during the following epochs:

![image](https://user-images.githubusercontent.com/6633976/187659373-00efdf50-3e02-407b-9df3-d997f0f81534.png)

I have seen this behaviour in all the trainings were I continue with a `best.pt` model from a previous run. Is there a reason to that?  Is there a way to prevent YOLOv5 to loose that much performance when I continue a training?

### Environment

- YOLOv5 🚀 v6.2-57-gf2b8f3fe
- Ubuntu 20.04
- Python Python-3.8.10
- torch-1.11.0+cu113

**Comment by glenn-jocher** [comment]
@haimat don't raise bug reports for questions.

**Comment by glenn-jocher** [comment]
👋 Hello! Thanks for asking about **resuming training**. YOLOv5 🚀 Learning Rate (LR) [schedulers](https://github.com/ultralytics/yolov5/blob/aa542ce6a65658ff931fee9bbab77c0145c152f0/train.py#L175-L181) follow predefined LR curves for the fixed number of `--epochs` defined at training start (`default=300`), and are designed to fall to a minimum LR on the final epoch for best training results. For this reason you can not modify the number of epochs once training has started.

<img width="850" alt="Screenshot 2022-04-10 at 11 11 51" src="https://user-images.githubusercontent.com/26833433/162611202-4e0b611f-6cca-4e43-94fc-bbc44e6447f8.png">

If your training was **interrupted** for any reason you may continue where you left off using the `--resume` argument. If your training fully completed, you can start a **new** training from any model using the `--weights` argument. Examples:

## Resume Single-GPU

You may not change settings when resuming, and **no additional arguments** other than `--resume` should be passed, with an optional path to the checkpoint you'd like to resume from. If no checkpoint is passed the most recently updated `last.pt` in your `yolov5/` directory is automatically found and used:
```bash
python train.py --resume  # automatically find latest checkpoint (searches yolov5/ directory)
python train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Resume Multi-GPU

[Multi-GPU DDP trainings](https://github.com/ultralytics/yolov5/issues/475) must be resumed with the same GPUs and DDP command, i.e. assuming 8 GPUs:
```bash
python -m torch.distributed.run --nproc_per_node 8 train.py --resume  # resume latest checkpoint
python -m torch.distributed.run --nproc_per_node 8 train.py --resume path/to/last.pt  # specify resume checkpoint
```

## Start from Pretrained

If you would like to **start training** from a fully trained model, use the `--weights` argument, not the `--resume` argument:
```bash
python train.py --weights path/to/best.pt  # start from pretrained model
```

Good luck 🍀 and let us know if you have any other questions!
